### PR TITLE
Make str_index available for constant str.

### PR DESCRIPTION
**Pull request by redbopo**
fix [issue-454](https://github.com/lcompilers/lpython/issues/454).

### CODE REVIEW

File: integration_tests/test_str_01.py [hunk 1]
```diff
@@ -45,11 +45,15 @@ def test_str_repeat():
     assert 3*a*3 == "XyzXyzXyzXyzXyzXyzXyzXyzXyz"
     assert a*-1 == ""
 
+def test_cosntant_str_index():
+    assert "abc"[2] == "c"
+
```
Comment (namannimmo10) [inline integration_tests/test_str_01.py:50]:
```suggestion
def test_constant_str_subscript():
    assert "abc"[2] == "c"
    assert "abc"[:2] == "ab"
```

File: integration_tests/test_str_01.py [hunk 2]
```diff
@@ -45,11 +45,15 @@ def test_str_repeat():
...
 def check():
     f()
     test_str_concat()
     test_str_index()
     test_str_slice()
     test_str_repeat()
+    test_cosntant_str_index()
```
Comment (namannimmo10) [inline integration_tests/test_str_01.py:57]:
```suggestion
    test_constant_str_subscript()
```

**Comment by namannimmo10** [comment]
Thanks for the fix, @redbopo.  Would you mind making these changes? We can merge this afterward.

**Comment by certik** [comment]
@redbopo thanks for the PR! And welcome. Thanks @namannimmo10 for the review.

**Comment by redbopo** [comment]
Thanks  @namannimmo10  and @certik , just fix from the suggestion.

**Comment by certik** [comment]
Thanks! The test failures are unrelated to this PR, I fixed them in #692.

Btw, next time you can just click on `Commit suggestion` and it would commit Naman's suggestion.

Once #692 is in, I'll rebase this and we can merge it.

**Review by certik — APPROVED** [review]
Thanks! This is ready to go in.

**Comment by redbopo** [comment]
Thanks certik, `Commit suggestion` sounds like an easy method.

**Pull request merged**